### ISSUE TITLE
Dependent pull requests

### ISSUE THREAD

**Issue by lohxx** [isaacs/github#1500]
Is it possible make a pull request depends on another? the pull request is only accepted if the pull that he depends on is merged.

**Comment by amcsi** [comment]
No, it's not, unfortunately. These issues cover that by the way: #959, #867

**Comment by donhatch** [comment]
dup of https://github.com/isaacs/github/issues/959 ?
